lib/screens/v2/widget_instance/elevator_status.ex

```diff
@@ -270,18 +283,25 @@ defmodule Screens.V2.WidgetInstance.ElevatorStatus do
         serialize_closure(alert, facility, now)
       end)
 
+    icons =
+      station_id_to_icons
+      |> Map.fetch!(parent_station_id)
+      |> Enum.take(3)
```

Comment:
out of curiosity, are there any situations we know of where the icon list is longer than 3?

Reply:
Hmm, good question. I did not encounter one in testing, but I threw the `take` on there just in case because design gave that maximum.